server/api/users/users.test.ts

```diff
@@ -36,4 +37,65 @@ describe('auth router', () => {
       });
     });
   });
+
+  describe('PATCH /:id', () => {
+    it('returns 403 if profile id is not equal to logged in user id', async () => {
+      const user = await createUser();
+
+      server.login(user);
+
+      const res = await server.exec.patch('/api/users/54321');
+      expect(res.status).toBe(403);
+    });
+
+    it('returns 400 if request body does not contain any updatable fields', async () => {
+      const user = await createUser();
+
+      server.login(user);
+
+      const res = await server.exec.patch(`/api/users/${user.id}`).send({ asdf: 'asdf' });
```

Comment:
May also be worth testing that the appropriate response is given when _no data_ is provided.